pkg/volume/testing/testing.go

```diff
@@ -1530,3 +1533,132 @@ func (f *fakeVolumeHost) SetKubeletError(err error) {
 func (f *fakeVolumeHost) WaitForCacheSync() error {
 	return nil
 }
+
+type FakeOperationStartTimeCache struct {
```

Comment:
An alternative to make this public might be adding Keys() method on the actual interface.